### ISSUE TITLE
allow users to update their email address

### ISSUE THREAD

**Issue by josh-chamberlain** [Police-Data-Accessibility-Project/data-sources-app#380]
## Context

follows #318 

### Back end

- [ ] allow users to update their own email address with an endpoint

### Front end

- [ ] allow users to edit their email address in their user profile

**Comment by maxachis** [comment]
@josh-chamberlain Should we consider this a v2 thing?

**Comment by josh-chamberlain** [comment]
@maxachis I don't think so. This doesn't need to be self serve for the purposes of the MVP. It's an edge—people can email us.
